test/dotnet-sln-add.Tests/GivenDotnetSlnAdd.cs

```diff
@@ -839,29 +887,28 @@ public void WhenPassedMultipleProjectsAndOneOfthemDoesNotExistItCancelsWholeOper
                 .Should().BeVisuallyEquivalentTo(contentBefore);
         }
 
-        //ISSUE: https://github.com/dotnet/sdk/issues/522
-        //[Fact]
+        [Fact(Skip = "https://github.com/dotnet/sdk/issues/522")]
         public void WhenPassedAnUnknownProjectTypeItFails()
         {
```

Comment:
It looks like this method was reformatted with a tab with of 3 rather than 4.

Reply:
Weird. Thanks for spotting this! I fixed it in 6038d3d38.